apps/api/src/services/data-provider/yahoo-finance/yahoo-finance.service.ts

```diff
@@ -136,6 +138,49 @@ export class YahooFinanceService implements DataProviderInterface {
     }
   }
 
+  public async search(aSymbol: string): Promise<{ items: LookupItem[] }> {
+    let items = [];
+
+    try {
+      const get = bent(
+        `https://query1.finance.yahoo.com/v1/finance/search?q=${aSymbol}&lang=en-US&region=US&quotesCount=8&newsCount=0&enableFuzzyQuery=false&quotesQueryId=tss_match_phrase_query&multiQuoteQueryId=multi_quote_single_token_query&newsQueryId=news_cie_vespa&enableCb=true&enableNavLinks=false&enableEnhancedTrivialQuery=true`,
```

Comment:
You might want to extract the hostname to a field to make testing with a mock HTTP server possible.

Reply:
✅